__DEFINES/setup.dm

```diff
@@ -1129,16 +1129,50 @@ var/default_colour_matrix = list(1,0,0,0,\
 #define MT_UPDATE 1
 #define MT_REINIT 2
 
+//////////////////
+// FIRE
+//////////////////
+
+// specific heats for burning objects, assuming perfect combustion (MJ/kg)
```

Comment:
is this supposed to be specific heat capacity, in J/kgK?
or fuel burn value?

if B please name it `fuel_value` or literally anything else because it's extremely similar to the chemistry specheatcaps, which have been notoriously retarded until the frenchman fixed them recently

Reply:
It's actually the heat of combustion which is measured in MJ/kg. I'll rename it accordingly so we don't get it mixed up with specific heat capacity.